pkg/rc/replication_controller.go

```diff
@@ -396,7 +401,8 @@ func (rc *replicationController) ensureConsistency(current types.PodLocations) e
 		}
 
 		rc.logger.WithField("node", pod.Node).WithField("intentManifestSHA", intentSHA).Info("Found inconsistency in scheduled manifest")
-		if err := rc.schedule(ctx, pod.Node); err != nil {
+
+		if err := rc.scheduleNoAudit(ctx, pod.Node); err != nil {
```

Comment:
potentially we could just do audit logging in `ensureConsistency()` as well. it's going to produce an extra audit log record unnecessarily which is a downside but the upside would mean we don't need this `scheduleNoAudit()` function and we also don't need the `Context()` getter on the auditing transaction, and it would prevent someone from doing `transaction.Commit(txn.Context()` in the wrong place (and thus not creating an audit log record)